notebooks/WFC3/ir_scattered_light_calwf3_corrections/requirements.txt

```diff
@@ -1,9 +1,10 @@
-astropy==5.3.3
-astroquery==0.4.6
-drizzlepac==3.5.1
-ginga==4.0.1
-matplotlib==3.7.0
-numpy==1.23.4
-stwcs==1.7.2
-wfc3tools==1.4.0
-crds==11.17.9
+astropy>=5.3.3
+astroquery>=0.4.6
+drizzlepac>=3.5.1
+ginga==4.0.1  # The notebook needs ginga.util.zcale
```

Comment:
does this need to be pinned or can it be a minimum?

Reply:
I have noticed that the `Ginga. scale` API has been changed, which is why I had to pin it to this version. I will attempt to update some of the notebooks, as long as they do not involve an understanding of scientific nuances, and replace them with the `astropy.visualization.ZScaleInterval`. For these updated notebooks, I will also contact the authors to get their input.